src/components/notifications/AppNotifications/AppNotificationDropdown/index.tsx

```diff
@@ -23,12 +22,10 @@ const AppNotificationDropdown: React.FC<IAppNotificationDropdownProps> = ({
   h,
   closeDropdown
 }) => {
-  const { notifyClientProxy } = useContext(W3iContext)
-
   const handleUnsubscribe = useCallback(() => {
     closeDropdown()
     unsubscribeModalService.toggleModal(notificationId)
-  }, [notificationId, closeDropdown, unsubscribeModalService, notifyClientProxy])
+  }, [notificationId, closeDropdown, unsubscribeModalService])
```

Comment:
Same for the previous memoization comment.

Reply:
As discussed, let's keep those for separate PR